README.md

```diff
@@ -4,30 +4,312 @@
+
+# Run with your configuration
+docker run -v /path/to/your/config:/etc/fedmsg.d/ \
+           -e GITHUB_TOKEN=your_token \
+           -e JIRA_TOKEN=your_jira_token \
+           quay.io/redhat-aqe/sync2jira:latest
+```
+
+### From Source
+
+```bash
+# Clone the repository
+git clone https://github.com/release-engineering/Sync2Jira.git
+cd Sync2Jira
+
+# Install dependencies
+pip install -r requirements.txt
+
+# Install the package
+pip install .
+```
 
 ## Configuration
 
-We have set up a quick-start [here](https://sync2jira.readthedocs.io/en/master/quickstart.html)
+Create a configuration file in the `fedmsg.d/` directory. Here's a basic example:
+
+```python
+# fedmsg.d/sync2jira.py
+config = {
+    'sync2jira': {
+        # GitHub API token
+        'github_token': 'your_github_token',
+        
+        # JIRA configuration
+        'default_jira_instance': 'primary',
+        'jira': {
+            'primary': {
+                'options': {
+                    'server': 'https://your-jira.example.com',
+                    'verify': True,
+                },
+                'token_auth': 'your_jira_token',
+            },
+        },
+        
+        # Repository mappings
+        'map': {
+            'github': {
+                'username/repository': {
+                    'project': 'JIRA_PROJECT_KEY',
+                    'component': 'component-name',
+                    'sync': ['issue', 'pullrequest'],
+                    'issue_updates': [
+                        'comments',
+                        'title',
+                        'description',
+                        {'tags': {'overwrite': True}},
+                        {'assignee': {'overwrite': False}},
+                        'url'
+                    ],
+                },
+            },
+        },
+        
+        # Optional settings
+        'testing': False,  # Set to True for dry-run mode
+        'initialize': False,  # Set to True to sync all existing issues
+        'admins': ['admin_username'],  # Email notifications
+    }
+}
+```
+
+### Configuration Options
+
+| Option | Description | Default |
+|--------|-------------|---------|
+| `github_token` | GitHub API token for authentication | Required |
+| `jira` | JIRA instance configurations | Required |
+| `map` | Repository to project mappings | Required |
+| `testing` | Enable dry-run mode (no actual changes) | `False` |
+| `initialize` | Sync all existing issues on startup | `False` |
+| `filters` | Filter issues by labels, milestones, etc. | `{}` |
+| `admins` | List of admin users for notifications | `[]` |
+
+## Usage
+
+### Running the Service
+
+```bash
+# Start the sync service
+sync2jira
+
+# With initialization (sync all existing issues)
+INITIALIZE=1 sync2jira
+
+# Testing mode (dry run)
+# Set testing: True in config file
+sync2jira
+```
+
+### Manual Sync Interface
+
+Access the sync page at `http://your-sync-page-url` to manually trigger synchronization for specific repositories.
```

Comment:
How is the reader supposed to know what `your-sync-page-url` means?

E.g., if they are running it locally, presumably that is `localhost`, but it probably needs a port number (i.e., it's probably _not_ `80`, because that requires privilege to bind to).

Also, I think that running `sync-page` requires a different invocation -- otherwise, you end up running the `sync2jira` listener instead of the micro-web-server.